go.mod

```diff
@@ -55,23 +55,26 @@ replace (
 )
 
 require (
+	github.com/Microsoft/hcsshim v0.8.18
 	github.com/containerd/continuity v0.0.0-20210208174643-50096c924a4e
 	github.com/google/go-containerregistry v0.5.0
+	github.com/google/gopacket v1.1.19
+	github.com/iamacarpet/go-win64api v0.0.0-20210311141720-fe38760bed28
 	github.com/k3s-io/helm-controller v0.10.1
 	github.com/pkg/errors v0.9.1
-	github.com/rancher/k3s v1.21.1-rc1.0.20210629142838-04398a2582a3
+	github.com/rancher/k3s v1.21.1-rc1.0.20210709172249-238dc2086e94
```

Comment:
As a team, we've been trying to be more careful about what we pull in when updating K3s in RKE2, and create issues in RKE2 for each feature or bugfix that gets pulled in so that QA can test. As part of this, we've started linking the commit log between the old and new release when updating k3s in go.mod:
https://github.com/k3s-io/k3s/compare/04398a2582a3...238dc2086e94

Reply:
To be clear - I'm not saying we shouldn't bump K3s - that needs to happen periodically and it's fine. We just need to look over the changes that are in that commit range and ensure that we create issues in RKE2 to track anything that QA should test.

Reply:
https://github.com/rancher/rke2/issues/1292